src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -6963,7 +7000,6 @@ test_client_side_encryption_install (TestSuite *suite)
                       NULL /* dtor */,
                       NULL /* ctx */,
                       test_framework_skip_if_no_client_side_encryption,
-                      test_framework_skip_if_max_wire_version_less_than_17,
```

Comment:
What is the rationale for using the new `SKIP_IF_NO_QUERYABLE_ENCRYPTION_V2` instead of `test_framework_skip_if_max_wire_version_less_than_<wire-version-for-7.0>`? Is a wire version number for server 7.0 not assigned yet?